lib/pundit.rb

```diff
@@ -233,14 +233,17 @@ def policy(record)
   #   If omitted then this defaults to the Rails controller action name.
   # @return [Hash{String => Object}] the permitted attributes
   def permitted_attributes(record, action = params[:action])
-    param_key = PolicyFinder.new(record).param_key
     policy = policy(record)
     method_name = if policy.respond_to?("permitted_attributes_for_#{action}")
       "permitted_attributes_for_#{action}"
     else
       "permitted_attributes"
     end
-    params.require(param_key).permit(*policy.public_send(method_name))
+    pundit_params_for(record).permit(*policy.public_send(method_name))
```

Comment:
@pablocrivella we still need to pass the Rails controller specific params implementation in, right? I don't think `Hash` has a `permit` instance method.

Reply:
Ok, should work then. :)